web/src/js/backends/websocket.tsx

```diff
@@ -35,9 +35,9 @@ export default class WebsocketBackend {
     }
 
     onOpen() {
+        this.fetchData("options")
         this.fetchData("flows")
         this.fetchData("events")
-        this.fetchData("options")
```

Comment:
Doesn't that introduce a new race condition on which request finishes first?

Reply:
Shall I make `connect()` run after `/options` is completed? I don't quite get it here, any advice from you?